src/System.Security.Principal.Windows/src/System/Security/Principal/NTAccount.cs

```diff
@@ -0,0 +1,393 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32;
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.Runtime.InteropServices;
+
+namespace System.Security.Principal
+{
+    public sealed class NTAccount : IdentityReference
+    {
+        #region Private members
+
+        private readonly string _Name;
```

Comment:
Some of these private fields are named incorrectly according to our naming guidelines, e.g. this should be ```_name``` rather than ```_Name```.